CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing to wolfi-dev/os
+
+<!-- toc -->
+- [Setup development environment](#setup-development-environment)
+- [Write your first Wolfi package](#write-your-first-wolfi-package)
+- [Package versioning](#package-versioning)
+- [Some tips](#some-tips)
+<!-- /toc -->
+
+## Setup development environment
+
+To ease the development of Wolfi OS, you can use `chainguard-images/sdk` container image that already includes both [apko](https://github.com/chainguard-dev/apko) & [melange](https://github.com/chainguard-dev/melange).
+
+To start a development environment, do
+
+```sh
+make dev-container
+```
+
+What it does is start the `chainguard-images/sdk` image and mount the current working directory into it.
+
+Now, the `Makefile` script assume that there're melange folder & melange binary the upper level dir (`../melange`). I change it locally to the following because it's where they are in `chainguard-image/sdk`.
+
+```
+MELANGE_DIR ?= /usr/share/melange
+MELANGE ?= $(shell which melange)
+```
+
+## Write your first Wolfi package
+
+Wolfi packages are built using melange. If you want to learn how packages are built, you can see all the details in the [`ci-build`](.github/workflows/ci-build.yaml) workflow and in the [Makefile](Makefile).
+
+Start by cloning this repository and create a YAML file named `<your-package-name>.yaml` in its root directory. If you have any patches, create a folder with the same name and put them there.
```

Comment:
>Start by cloning this repository

We might want to put this "cloning" instruction in the first section, since that would be the first place in reading this doc that the repo needs to have been cloned. WDYT?